index.js

```diff
@@ -102,6 +102,12 @@ function start() { // eslint-disable-line max-statements
     });
   }
 
+  // if worker dynos are not enabled run the Sample timeout query here.
+  if (conf.enableWorkerDynos === false || conf.enableWorkerDynos === 'false') {
```

Comment:
in general, I think we should put all of the conditions for env vars in config.js so that the modules which use them can just do a simple `if (conf.enableWorkerDynos)...` or `if (!conf.enableWorkerDynos)...`